### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by ashuralyk**

### CODE REVIEW

**Review by Flouse — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+
+```toml
+ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics", branch = "main" }
+```
+
+## Object Encoding/Decoding
+
+IBC-compatible objects owned by IBC cells are encoded with RLP algorithm, to ensure compatibility with Axon — the exclusively compatible counterparty chain so far, facilitating the integration with Solidity.
+
+Given the widespread adoption of the RLP encode/decode algorithm across mainstream programming languages, RLP can be considered as a standard algorithm for Forcerelay to encode and decode IBC-compatible objects, even in future cross-chain interactions between CKB/Axon and Cosmos.
```
Comment (Xcodes-chain) [inline README.md:21]: ckb uses 32bytes fixed-length binary, while geth(rlp) uses text. In the future, you can consider using ckb-auth-cli to convert messages used by geth (rlp codec) into ckb-auth (ckb) format.

- https://github.com/joii2020/ckb-auth/tree/ethereum

**Review by 15168316096 — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (ashuralyk) [reply README.md:21]: I guess ckb-auth is just used to unify all of different lock scripts, but the RLP mentioned here isn't related to the lock script, it's just a format of encoded bytes located in the `witnesses` field in a CKB transaction with its 32 bytes hash located in the `data` field of a cell in this transaction.

**Pull request merged**